### DISCUSSION TITLE
why not use vite to build a front-end vue project?

### DISCUSSION THREAD

**Discussion by eighteenzheng** [misitebao/wails-template-vue#3] [Q&A · 问答] [answered]
@misitebao 
Why not use vite to build front-end vue projects, vite builds faster, can improve development efficiency and improve development experience.

**Comment by misitebao** [comment, marked as answer]
> @misitebao
> Why not use vite to build front-end vue projects, vite builds faster, can improve development efficiency and improve development experience.

Because the wails front-end rendering engine (webview2 is used on windows) does not support esmodule.

**Reply by misitebao** [reply]
Now it can perfectly support esmodule, so we use vite to refactor the front-end part, you can experience flying speed.

**Reply by eighteenzheng** [reply]
Wow, that's great.
